src/main/java/se/michaelthelin/spotify/SpotifyHttpManager.java

```diff
@@ -128,6 +135,10 @@ public static URI makeUri(String uriString) {
     }
   }
 
+  public HttpClientConnectionManager getConnectionManager() {
```

Comment:
getConnectionManager is no longer necessary

Reply:
We have those getters for all class variables right now, like `getProxy`, ... which we don't use in our code base either, so I just added this as well.